### PR TITLE
feat: update staking pool tests

### PR DESCRIPTION
**Pull request by hejkerooo**
- fixed staking-pool tests with claim manager integration,
- disabled tests for not used staking,

### CODE REVIEW

File: src/init.ts [hunk 1]
```diff
@@ -64,7 +64,7 @@ export async function init(signerService: SignerService) {
             const claimsService = await ClaimsService.create(signerService, domainsService, cacheClient, didRegistry);
             return { didRegistry, claimsService };
         }
-        return { cacheClient, domainsService, stakingService, assetsService, connectToDidRegistry, stakingPoolService };
```
Comment (Harasz) [inline src/init.ts:67]: Is this intentional?

File: e2e/staking-pool.e2e.ts [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+        await signerService.connect(patron, ProviderType.PrivateKey);
+        await replenish(orgOwner.address, "50.0");
+        await replenish(patron.address, "50.0");
+        await timeTravel(provider, 60);
+
+        stakingPoolFactoryService = new StakingFactoryService(signerService, domainsService);
+    });
+
+    it("can stake", async () => {
+        expect(true).toBe(true);
```
Comment (JGiter) [inline e2e/staking-pool.e2e.ts:217]: Left from debug?

**Review by JGiter — CHANGES_REQUESTED** [review]

File: e2e/staking-pool.e2e.ts (hunk 2 above)
Comment (hejkerooo) [reply e2e/staking-pool.e2e.ts:217]: yeah, will remove later, thx

**Comment by Harasz** [comment]
@hejkerooo What is the status of this PR?

**Pull request closed without merging**